Packs/CyrenInboxSecurity/Playbooks/Cyren_Inbox_Security_Default_README.md

```diff
@@ -0,0 +1,29 @@
+Processes Cyren Incidents, sets resolutions, and applies remediations to end-user mailboxes. 
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+* CyrenInboxSecurity
+
+### Scripts
+This playbook does not use any scripts.
+
+### Commands
+* closeInvestigation
+* cyren-resolve-and-remediate
+
+## Playbook Inputs
+---
+There are no inputs for this playbook.
+
+## Playbook Outputs
+---
+There are no outputs for this playbook.
+
+## Playbook Image
+---
+![Cyren Inbox Security Default](../doc_files/Cyren_Inbox_Security_playbook_1.png)
```

Comment:
Please make sure that the name of the playbook ang his image are same

Reply:
done